community/graphdb-api/src/main/java/org/neo4j/graphdb/schema/ConstraintDefinition.java

```diff
@@ -28,7 +28,7 @@
  * <b>Note:</b> This interface is going to be changed/removed in next major release to better cope with node and
```

Comment:
Aside from the fact that the comment is not introduced by our code, I actually think there is a reasonable chance that our next wave of refactoring might cause the comment to come true, or at least partly true. We could remove this, or just wait for the composite index work to deal with it.